OneCommunityGlobal/HighestGoodNetworkApp
# aaron linting for PermissionsManagement component

# Description
Fix linting for the `PermissionsManagement` component

## Related PRS (if any):
N/A

## Main changes explained:
- removed the PermissionsManagement component directory from `.eslintignore` and `.prettierignore`

## How to test:
1. check into current branch
2. do `npm install` and `npm run lint` to make sure you get no lint errors
3. run `npm run start:local` and start the backend as usual
4. go to Other Links -> Permissions Management and make sure everything is working (please do not add/remove any permissions).
5. run `npm run test` to make sure the tests are passing (see below image)

![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/10265513/8a934975-4cc6-460e-a342-5e2baad311d2)

## Note:
- the failing tests are also on dev branch, I am looking into that
- there were no lint errors, so I haven't really changed any of the component's code so nothing should be breaking
